distribution/src/bin/elasticsearch-service.bat

```diff
@@ -121,6 +121,7 @@ rem   - third, JVM options from ES_JAVA_OPTS are applied
 rem   - fourth, ergonomic JVM options are applied
 
 if not "%ES_JAVA_OPTS%" == "" set ES_JAVA_OPTS=%ES_JAVA_OPTS: =;%
+if not "%ES_JAVA_OPTS%" == "" set ES_JAVA_OPTS=%ES_JAVA_OPTS:;;=;%
```

Comment:
It it normal to duplicate the line with the new one instead of replacing the older one (the L123?)